pkg/operator/resources/cluster/cdiconfig.go

```diff
@@ -28,6 +28,81 @@ func createCDIConfigCRD() *extv1beta1.CustomResourceDefinition {
 			},
 			Version: "v1alpha1",
 			Scope:   "Cluster",
+			Validation: &extv1beta1.CustomResourceValidation{
+				OpenAPIV3Schema: &extv1beta1.JSONSchemaProps{
+					Description: "CDIConfig is the configuration object for Containerized Data Importer",
+					Type:        "object",
+					Properties: map[string]extv1beta1.JSONSchemaProps{
+						"apiVersion": {
```

Comment:
According to this, can leave out apiVersion, kind, metadata:  https://kubernetes.io/blog/2019/06/20/crd-structural-schema/

Reply:
Done